xrmocap/data_structure/body_model/smpl_data.py

```diff
@@ -60,14 +55,16 @@ def __init__(self,
                 in shape [n_frame, betas_dim].
                 Defaults to None,
                 zero-tensor in shape [n_frame, 10] will be created.
+            mask (Union[np.ndarray, torch.Tensor, None], optional):
```

Comment:
`mask` is frame-level mask?

Reply:
Yes. Each smpl data holds data for only one person, and we do not mask parts or joints.

Reply:
Shall we doc it as `A tensor or ndarray for frame-level visibility mask` or rename it to `frame_mask`?

Reply:
We won't have another mask for joints or parts, will we? Considering the parents and children of joints, it is not possible to mask an independent joint.
By setting a keypoint's visibility to 0, we directly remove it from visualization. But for a joint, we can only set it to zero and make it look like an initiative pose, visibility has not been changed.

Reply:
Docstring fixed in [fb637eb](https://github.com/openxrlab/xrmocap/pull/73/commits/fb637ebd1bc617ef874ee4f15dc0b6dd57ba3c8f).